lib/plugin.js

```diff
@@ -75,10 +91,28 @@ exports.register = function (server, opts, next) {
         patch: patch,
         delete: del
     })
-    
+
+    server.expose('schemas', schemas);
+
     server.ext('onPostStop', (server, next) => {
         adapter.disconnect(next)
     })
+
+    server.ext('onPreResponse', function (req, reply) {
+            if (req.query.include) {
+                Promise.resolve().then(function () {
+                    var type = req.path.match(/^\/([^/]+).*/)[1];
+                    return appendLinkedResources(req.response.source, type, req.query.include.split(','));
+                }).then(function (body) {
+                    reply(body);
+                }).catch(function (error) {
```

Comment:
fair enough, but we need to somehow let Hapi handle this. Maybe use ```finally``` instead of catch. Leave this for now let's see how Steve handles errors.